### ISSUE TITLE
Fix false positives for Less variables and mixins in at-rule-semicolon-newline-after

### ISSUE THREAD

**Issue by pvdlg** [stylelint/stylelint#3764]
> Clearly describe the bug

Similar to issue https://github.com/stylelint/stylelint/issues/3749 with rule `at-rule-semicolon-newline-after` which wasn't included in the fix https://github.com/stylelint/stylelint/pull/3759/

> Which rule, if any, is the bug related to?

`at-rule-semicolon-newline-after`

> What CSS is needed to reproduce the bug?

```css
@some-color: #000;

@some-other-color: #111;
```
> Which version of stylelint are you using?

`9.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint "**/*.less"`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Less variables.

> What did you expect to happen?

No warnings to be flagged.

**Comment by jeddy3** [comment]
@pvdlg Thanks for the report and for using the template.

Did we miss any other `at-rule-*` rules?

**Comment by pvdlg** [comment]
Not in my project were I saw this issue. But I didn't tested all the rules.
